src/content/developers/docs/scaling/zk-rollups/index.md

```diff
@@ -41,9 +229,35 @@ Multiple implementations of ZK-rollups exist that you can integrate into your da
 
 <RollupProductDevDoc rollupType="zk" />
 
-**ZK-rollups reading**
+## Aztec Network 
```

Comment:
Im going to remove these and add the information in a separate PR for adding Aztec and Hermez. That way they will be apart of the RollupProductDevDoc list :)